### PR TITLE
Extend log tests

### PR DESCRIPTION
**Pull request by tomwillis608**

### CODE REVIEW

File: tests/test_log.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+"""Test cases for the __main__ module."""
+import logging
+import os
+from unittest.mock import Mock
```
Comment (basiliskus) [inline tests/test_log.py:4]: Can be replaced by `pytest-mock`

File: tests/test_log.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+def test_setup_logging_repro_yaml():
+    """Test logging setup with test config from repro."""
+    setup_logging(verbose=False, debug=False)
+
+    assert os.path.exists(ROOT_PATH / "logs")
+    assert os.path.exists(ROOT_PATH / "logging.yaml")
+    assert len(logging.getLogger().handlers) == 2
+
+    console_handler = logging.getLogger().handlers[1]
+    assert isinstance(console_handler, logging.FileHandler)
```
[the remark this answers is not in the document]
Comment (tomwillis608) [reply tests/test_log.py:23]: that is probably a poorly named variable

File: tests/test_log.py (hunk 1 above)
Comment (tomwillis608) [reply tests/test_log.py]: I made that change

File: src/devops_deployment_metrics/log.py [hunk 3]
```diff
@@ -3,8 +3,7 @@
 import os
 
 import yaml
-
-from .definitions import ROOT_PATH
+from devops_deployment_metrics.definitions import ROOT_PATH
```
[the remark this answers is not in the document]
Comment (tomwillis608) [reply src/devops_deployment_metrics/log.py]: I made that change

File: tests/test_log.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
+"""Test cases for the __main__ module."""
+import logging
+import os
+from unittest.mock import Mock
+
+import pytest
+from devops_deployment_metrics.log import get_logger
+from devops_deployment_metrics.log import ROOT_PATH
```
[the remark this answers is not in the document]
Comment (tomwillis608) [reply tests/test_log.py]: I made that change

File: tests/test_log.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomwillis608) [reply tests/test_log.py]: I fixed the variable name

**Review by basiliskus — APPROVED** [review]

**Pull request merged**